Rakefile

```diff
@@ -1,22 +1,22 @@
 require 'rake'
 require 'rake/testtask'
 
-require_relative 'lib/tasks/exercise_test_task'
+require_relative 'lib/tasks/exercise_tests'
 
 task default: 'test'
 
-desc 'Run all development and exercise tests'
+desc 'Run individual exercises or run all development and exercise tests'
 task :test do
   exercises = ARGV.take_while { |e| e != '--' }.drop(1)
 
   if exercises.any?
-    exercises.each do |e|
-      task(e) { Rake::Task['test:exercise'].execute(exercise: e) }
-    end
-  else
-    Rake::Task['test:dev'].invoke
-    Rake::Task['test:exercises'].invoke
+    exercises.each { |e| task(e) { } }
```

Comment:
Is this something that anyone using rake should know?

If not putting it in an intention revealing method could be good:
```
create_null_tasks_for(exercises)
```